src/general/OptionsMenu.cs

```diff
@@ -162,6 +162,11 @@ public class OptionsMenu : Control
     [Export]
     public NodePath CustomUsernamePath;
 
+    // Sound tab cache
```

Comment:
I think this comment should just be removed, as there's just one variable here. If you want you could add an xml doc comment for this variable.

Reply:
Shoulde said so from the beginning........